examples/frontend/pages/_app.tsx

```diff
@@ -4,6 +4,7 @@ import { ThemeProvider } from 'theme-ui'
 import { theme } from '../theme'
 import { RecoilRoot } from 'recoil'
 import { NearHooksProvider } from '@cura/hooks'
+import '@cura/components/dist/assets/fonts/index.css'
```

Comment:
Isn't this better suited in `_document`?

Reply:
I get the following error when I do that
````
CSS cannot be imported within pages/_document.js. Please move global styles to pages/_app.js.
````